toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -219,6 +219,7 @@ libksba-devel-1.6.0-1.cm2.aarch64.rpm
 npth-1.6-4.cm2.aarch64.rpm
 pinentry-1.2.0-1.cm2.aarch64.rpm
 gnupg2-2.3.3-3.cm2.aarch64.rpm
+gnupg2-lang-2.3.3-3.cm2.aarch64.rpm
```

Comment:
Unclear to me if we need these in the worker chroot (admittedly better safe than sorry right now).

Reply:
I see glibc-lang rpm added in worker chroot hence adding for chkconfig, newt and gnupg2 lang paxkages too.

Reply:
ok, sounds good